tests/components/unmuteIcon-test.js

```diff
@@ -0,0 +1,135 @@
+jest.dontMock('../../js/components/unmuteIcon');
+jest.dontMock('../../js/constants/constants');
+jest.dontMock('classnames');
+
+var React = require('react');
+var ReactDOM = require('react-dom');
+var TestUtils = require('react-addons-test-utils');
+var UnmuteIcon = require('../../js/components/unmuteIcon');
+var CONSTANTS = require('../../js/constants/constants');
+var skinConfig = require('../../config/skin.json');
+var sinon = require('sinon');
+
+describe('UnmuteIcon', function () {
+  it('creates an expanded UnmuteIcon', function () {
+    var mockController = {
```

Comment:
Also not super important, but this could probably be defined in `beforeEach` in order to avoid defining it every time.

Reply:
Done